src/irmin-pack/unix/ext.ml

```diff
@@ -248,7 +248,8 @@ module Maker (Config : Conf.S) = struct
                 cancelled
             | None -> false
 
-          let start ~unlink ~use_auto_finalisation t commit_key =
+          let start ~unlink ~use_auto_finalisation ~path_new_files t commit_key
```

Comment:
Or `dst_root`, but I'm fine with any

Reply:
FWIW, I first thought of proposing `output_path` but then realized that isn't accurate since the results file still writes to `root`. I think `dst_root` _maybe_ has a similar issue, depending on your perspective. I stuck with simply rearranging the existing name just because it is more common to put `_path` at the end when naming a path variable.